dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/context/ContextManager.java

```diff
@@ -236,6 +248,28 @@
     /** Write to secondary server configuration. */
     private boolean iWriteToSecondary = false;
 
+    /** The Ldap repo ID for use in logging **/
+    private String reposId = null;
+
+    /** Sets the bind authentication mechanism for administrative binds to the Ldap server. None, simple, GSSAPI (Kerbers). */
+    private String bindAuthMechanism = null;
+
+    /** GSSAPI/Kerberos configuration. krb5Principal is required, krb5TicketCache is optional. Keytab/config can be configured in the <kerberos> element */
+    private String krb5Principal = null;
+    private Path krb5TicketCache = null;
+
+    /** The KerberosService for use when bindAuthMechanism is GSSAPI (Kerberos), also loads the keytab/config if configured in the <kerberos> element */
+    private KerberosService kerberosService = null;
+
+    public ContextManager() {
```

Comment:
Do we need the empty default constructor?

Reply:
We did after I added the constructor with parameters.

Reply:
We need it for the unit tests unless we want to change all the unit tests to pass in null/null for my new constructor.

Reply:
Removed.